<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>guanqia_1</class>
 <widget class="QWidget" name="guanqia_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCommandLinkButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>700</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Algerian</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Algerian&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="goalLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;个数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="timesLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
